src/infra.rs

```diff
@@ -14,31 +15,41 @@ pub mod test {
     use anyhow::Result;
     use once_cell::sync::Lazy;
     use sqlx::{migrate::Migrator, Any, Pool};
-    use tokio::sync::OnceCell;
+    use tokio::{runtime::Runtime, sync::OnceCell};
 
+    // destroy runtime, destroy connection pool, so use as static in all test
+    // ref. https://qiita.com/autotaker1984/items/d0ae2d7feb148ffb8989
+    pub static TEST_RUNTIME: Lazy<Runtime> = Lazy::new(|| {
```

Comment:
in runtime end, sqlx connection pool is broken (from 0.7?). So one shared static Runtime is used in all tests.